committeeoversightapp/templatetags/committeeoversight_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+from slugify import slugify
```

Comment:
As we talked about today, you could eliminate a dependency here if you used Djangos' built in [`slugify()` function](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.text.slugify):

```suggestion
from django.utils.text import slugify
```